pkg/servicelb/controller.go

```diff
@@ -356,6 +356,22 @@ func (h *handler) newDaemonSet(svc *core.Service) (*apps.DaemonSet, error) {
 
 		ds.Spec.Template.Spec.Containers = append(ds.Spec.Template.Spec.Containers, container)
 	}
+
+	// Add toleration to noderole.kubernetes.io/master=*:NoSchedule
+	noScheduleToleration := core.Toleration{
+		Key:      "noderole.kubernetes.io/master",
```

Comment:
Idk if this is wrong but the key is node-role.kubernetes.io/master everywhere else - I stumbled over that because I try to deploy a k3s cluster with applied master role taint and traefik is not gonna get an external ip on the master node. 

@louiznk

Reply:
Ok, I see it's already fixed upstream

https://github.com/rancher/k3s/commit/1ed12cffa01124689ac261fc72fcbef347364e57

Reply:
> Ok, I see it's already fixed upstream
> 
> [1ed12cf](https://github.com/rancher/k3s/commit/1ed12cffa01124689ac261fc72fcbef347364e57)

yes, it's have already been fixed, thanks